### ISSUE TITLE
Feature Request: Add resizable windows and iPad support

### ISSUE THREAD

**Issue by dch82** [rileytestut/Delta#294]
I’m not sure if this is asking for too much, but I would like it very much to be able to have resizable windows so I can full screen on both my iPad and on the external Stage Manager display.

Thanks!

**Comment by asesh** [comment]
> I’m not sure if this is asking for too much, but I would like it very much to be able to have resizable windows so I can full screen on both my iPad and on the external Stage Manager display.
> 
> Thanks!

I agree with you. It would be great, if iPad was natively supported too.
